taxonomies/food/categories.txt

```diff
@@ -70088,19 +69880,15 @@ ciqual_food_name:fr: Fruit de la passion ou maracudja, pulpe et pépins, cru
 
 < en:Tropical fruits
 en: Pineapple
+xx: ananas
```

Comment:
I think it makes sense to use xx: when there is really a term that works almost universally. e.g. when it's a name that is very distinctive (because it's origially from one specific country), like Calvados, Whisky, Gnocchi, Spaghetti, Paella, Chili con carne, that is understood in most countries. But I would not use xx: for common nouns that have different translations / endings in different languages, like "ananas", "papaya", "mandarina".